sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -2232,6 +2232,7 @@ object SupportedOpsForTools {
         val formatLowerCase = format.toString.toLowerCase
         val formatEnabled = formatLowerCase match {
           case "csv" => conf.isCsvEnabled && conf.isCsvReadEnabled
+          case "delta" => conf.isDeltaEnabled
```

Comment:
we should file an issue to test delta with the qualification tool

Reply:
Filed #7294